js/bitflyer.js

```diff
@@ -226,7 +226,7 @@ module.exports = class bitflyer extends Exchange {
             'baseVolume': this.safeNumber (ticker, 'volume_by_product'),
             'quoteVolume': undefined,
             'info': ticker,
-        };
+        });
```

Comment:
Same story here, let's propagate the `market`.

Reply:
yup i've missed it but I wanted to add it 🤦